newrelic/resource_newrelic_synthetics_private_location.go

```diff
@@ -114,13 +115,15 @@ func resourceNewRelicSyntheticsPrivateLocationRead(ctx context.Context, d *schem
 
 	resp, err := client.Entities.GetEntityWithContext(ctx, guid)
 	if err != nil {
-		if _, ok := err.(*errors.NotFound); ok {
-			d.SetId("")
-			return nil
+		if err.Error() == "Argument \"guid\" has invalid value $guid." {
+			return diag.FromErr(fmt.Errorf("invalid GUID"))
```

Comment:
Can we add the value of the GUID that the user passed? There's been some complaints about this.

Reply:
Sure